logstash-core/lib/logstash/api/modules/base.rb

```diff
@@ -0,0 +1,41 @@
+require "logstash/api/app_helpers"
+require "logstash/api/command_factory"
+
+module LogStash
+  module Api
+    module Modules
+      class Base < ::Sinatra::Base
```

Comment:
Well, I think its fairly common to have a 'base' class that's a sibling to the classes that inherit from it.

If we move it up a level it'd have to be renamed to `LogStash::Api::BaseModule` and moved to `api/base_module.rb`.

I'm fine with either approach. I take it you prefer the latter?